src/components/Modals/FiatRamps/FiatRampsCommon.ts

```diff
@@ -16,6 +16,7 @@ export type FiatRampAsset = {
   symbol: string
   imageUrl?: string
   disabled?: boolean
+  isBelowTheMinimumUsdAmountWhenSelling?: boolean
```

Comment:
That's quite verbose, what about something like `isBelowSellThreshold` or `hasEnoughCryptoForSell` ?